### ISSUE TITLE
Motion sensor accessories reset when restarting Homebridge

### ISSUE THREAD

**Issue by ayanm00** [Brandawg93/homebridge-nest-cam#174]
**Describe the bug**
The motion sensor accessories are reset when Homebridge restarts. The names of all motion sensors are reset to the default names, and any automations triggered by the motion sensors are deleted.

**To Reproduce**
1. Rename a motion sensor accessory, or use it to trigger an automation in HomeKit

2. Restart Homebridge

3. The motion sensor returns to its default name and the automation is deleted

**Expected behavior**
The motion sensor accessory name should stay whatever I saved it to be, and the automation should not be deleted.

Please answer these questions before submitting this issue:
```
On what OS are you running homebridge: Tested on macOS 10.13.6 High Sierra and macOS 10.15.6 Catalina
On what iOS version are you having issues: iOS 13.6
What camera types are having issues: Nest Hello
Does this issue occur consistently or randomly: Consistently
If applicable, what was the last version that worked properly: 3.5.0
```

Let me know if you need any more information. Thanks!

**Comment by Brandawg93** [comment]
It sounds like your homebridge instance is clearing cache after every restart. After testing mine, the name change stays and the automations are still there.

**Comment by ayanm00** [comment]
Any tips on how to prevent homebridge from doing? This is only happening on the motion sensors. The camera from the doorbell isn’t resetting, and none of my other accessories are either. Thanks for the quick reply!

**Comment by Brandawg93** [comment]
Just to be sure this actually isn't a bug, I've created a test version with a few things changed. You can install it via `npm -g i homebridge-nest-cam@test --unsafe-perm`. Let me know if that helps.

**Comment by Brandawg93** [comment]
Also, if you updated from 3.5.0 to 4.4.x, you may want to remove your cameras from your homebridge cache manually, and restart homebridge. It could be that you are seeing old versions of the motion sensors since they were reworked in 4.x.

**Comment by ayanm00** [comment]
The test version appears to have fixed the issue. I'll let you know if anything else happens. Thanks for your help!

**Comment by ayanm00** [comment]
Also, I have another quick question. What is considered an "important event"?

**Comment by Brandawg93** [comment]
Good to hear the test version fixed it! I'll close this whenever I officially release it so others can see this in the notes.

I'm not really sure what Nest considers an "important event". I just know that Nest has an attribute on motion events called "is_important". If you watch your cameras on https://home.nest.com, some of the event bubbles will be greyed out. Those are considered "not important".

**Comment by Brandawg93** [comment]
Just a heads up, I've created a new test version with a lot of changes to it. If you're able, could you try it out just to make sure the previous fix still works for you? I just want to make sure this fix wasn't reverted.

**Comment by ayanm00** [comment]
Just updated to the new test version, and the issue came back.

**Comment by Brandawg93** [comment]
And it's only for the motion sensors?

**Comment by ayanm00** [comment]
Yep, the camera name doesn’t return to the default one.

**Comment by Brandawg93** [comment]
Do you mean the camera name *does* return to the default? And are automations still working?

**Comment by ayanm00** [comment]
Basically the same issue as I described in the original post came back. Automations disappeared, the motion sensor names returned to their default values, but the camera name stayed as what I changed it to.

**Comment by Brandawg93** [comment]
Ok. What happens if you install the previous test? `npm install homebridge-nest-cam@4.4.3-test.0 --unsafe-perm`?

**Comment by ayanm00** [comment]
It works as expected on the previous test. Nothing is reset when homebridge is restarted.

**Comment by Brandawg93** [comment]
Interesting. Good to know. I'll do more testing and see what the differences are. I'll report back tomorrow to let you test it if you have time. Thanks for helping me test this!

**Comment by ayanm00** [comment]
Okay, sounds good. No problem! Thanks for making such a great plugin!

**Comment by lukescott** [comment]
Could it be that `is_important` is anything checked under "Home Notifications". So if you only want notifications within a motion zone you check only those boxes and `is_important` is true only for that zone?

**Comment by Brandawg93** [comment]
@ayanm00 I've asked around and other Mac OS users are experiencing the same issue. I've created a new test that may fix it. You can install it via `npm -g i homebridge-nest-cam@test --unsafe-perm`. Let me know if this one works.

**Comment by ayanm00** [comment]
@Brandawg93 The latest test version does not have the issue that I was experiencing.

**Comment by Brandawg93** [comment]
This fix is in v4.5.0. Thanks for the help testing!
